### PR TITLE
Add siteimprove analytics script

### PR DESCRIPTION
**Pull request by Arth101**
Trello: https://trello.com/c/DmzIfTYH/122-tilf%C3%B8je-siteimprove-analytics

### CODE REVIEW

File: shared/views/includes/analytics.pug [hunk 1]
```diff
@@ -6,3 +6,4 @@ script(type='text/javascript')
   if config.google.analyticsPropertyID
     | ga('create', '#{config.google.analyticsPropertyID}', 'auto');
   | ga('send', 'pageview');
+script(async, src="https://siteimproveanalytics.com/js/siteanalyze_#{config.siteimprove.analytics.id}.js")
```
Comment (hypesystem) [inline shared/views/includes/analytics.pug:9]: Would be nice to wrap in an if-statement, so we only render the script if the ID is defined

**Review by hypesystem — APPROVED** [review]
Looks good, if you test it and it works :smile: and check that we don't have any pseudo page loads in the codebase (places where we manually trigger a `ga('send','pageview')` other than the primary page load

File: shared/views/includes/analytics.pug (hunk 1 above)
Comment (Arth101) [reply shared/views/includes/analytics.pug]: I agree. I have added it now :)

**Review by hypesystem — COMMENTED** [review]
✅

**Pull request merged**